Containerfile.aws-load-balancer-operator-bundle

```
@@ -32,12 +32,32 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=aws-load-balancer-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=stable-v1.3,stable-v1
+LABEL operators.operatorframework.io.bundle.channels.v1=stable-v1.2,stable-v1
 LABEL operators.operatorframework.io.bundle.channel.default.v1=stable-v1
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.16.0+git
 LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
 LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
 
+# should this operator be supported on OCP 4.4 and earlier (old appregistry format)
+LABEL com.redhat.delivery.backport: false
+
+# this is a bundle image and should be delivered via an index image
+LABEL com.redhat.delivery.operator.bundle: true
+
+# Define build arguments to receive dynamic values
+ARG FULL_COMMIT
+ARG GIT_URL
+
+LABEL name=aws-load-balancer-operator-bundle
+LABEL version="1.2.0"
```

```suggestion
LABEL version="1.3.0"
```
As per the other comment.